deploy/operator.yaml

```diff
@@ -298,7 +298,7 @@ spec:
             fieldRef:
               fieldPath: metadata.annotations['olm.targetNamespaces']
         - name: KUBEVIRT_VERSION
-          value: v0.40.0-rc.2
+          value: v0.41.0-rc.1
         - name: VIRT_API_SHASUM
           value: sha256:0e0ae908962d132d6190e7e956d19b7a93d90b2e9fad32729c49dc39852a0899
```

Comment:
this smells like a bug in how we generate this file,
I'd expect sha256:b20cd432b45097be16565dedd5620b74d83185d044b4c2fa3a106b958b0d1d86 as in the CSV

Reply:
resolved removing pinned images from hack/build-manifests.sh